pkg/filters/wasmhost/vm.go

```diff
@@ -30,19 +30,20 @@ import (
 
 // WasmVM represents a wasm VM
 type WasmVM struct {
-	host    *WasmHost
-	ctx     *context.Context
-	store   *wasmtime.Store
-	inst    *wasmtime.Instance
-	ih      *wasmtime.InterruptHandle
+	host  *WasmHost
+	ctx   *context.Context
+	store *wasmtime.Store
+	inst  *wasmtime.Instance
+	//https://github.com/bytecodealliance/wasmtime/pull/3925
+	//ih      *wasmtime.InterruptHandle
```

Comment:
please remove these 2 historical lines of code, code changes can be tracked by git.

```suggestion
	//https://github.com/bytecodealliance/wasmtime/pull/3925
	//ih      *wasmtime.InterruptHandle
```